config/config.exs

```diff
@@ -1,19 +1,21 @@
 import Config
 
-config :prima_auth0_ex,
-  auth0_base_url: "https://tenant.eu.auth0.com"
-
+# Default client, for backwards compatibility
 config :prima_auth0_ex, :client,
+  auth0_base_url: "https://tenant.eu.auth0.com",
```

Comment:
Isn't better to not put a value here since this url doesn't exist?

Reply:
Tests would still work, but in theory it's still required so I think **something** should be there... maybe a less "real" URL?

Reply:
Yeah, that works for me

Reply:
Done!

Reply:
Done!